cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -128,6 +129,14 @@ def _create_entrance_exam(request, course_key, entrance_exam_minimum_score_pct=N
     }
     CourseMetadata.update_from_dict(metadata, course, request.user)
 
+    # Create the entrance exam section item.
+    create_xblock(
+        parent_locator=unicode(created_block.location),
+        user=request.user,
+        category='sequential',
```

Comment:
It looks like there is still disagreement about the subsection name (comments below).

My opinion is that the subsection name should be visible on the outline because it is shown in 2 places on the unit page (top breadcrumb and RHS navigator). Since the user will be exposed to the name anyway, hiding it on the outline doesn't do anything and just makes the UI inconsistent.